nimble/host/services/auracast/syscfg.yml

```diff
@@ -0,0 +1,27 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+syscfg.defs:
+  BLE_SVC_AURACAST_ADV_INSTANCE:
+    description: >
+      Defines which advertising instance is used for Auracast.
+    value: 42
+
+syscfg.restrictions:
+  - BLE_PERIODIC_ADV: 1
```

Comment:
this likely doesn't work as expected

Reply:
I removed this all together, added 7abb8e66a5d672d94d2dde33ff294a8809e9abde to fetch non-configured advertising instance and Auracast will report back what was found. Maybe this commit should be moved to another PR